pint/testsuite/pandas_test_interface.py

```diff
@@ -146,7 +147,35 @@ class TestInterface(base.BaseInterfaceTests):
     pass
 
 class TestMethods(base.BaseMethodsTests):
-    pass
+
+    @pytest.mark.filterwarnings("ignore::pint.UnitStrippedWarning")
```

Comment:
I'd like to keep these warnings popping up unless there's a good reason to silence them?

Reply:
The ones I've been silencing should be occurring without going through the PintArray, so there's nothing we can do in pint_array.py to get rid of them - That'd have to be done in quantity.py.

Main reason I'd been silencing them was to get through them all easier. I think it's easier to spot whether a future change starts creating a UnitStrippedWarning like this.

I'm not fussed either way.

Reply:
ah ok makes sense, if they're not an issue then we leave it for now